requirements/dev.txt

```diff
@@ -89,8 +89,8 @@ snowballstemmer==2.0.0    # via pydocstyle
 sqlparse==0.3.1           # via django
 stevedore==1.32.0         # via edx-opaque-keys
 text-unidecode==1.3       # via faker
-toml==0.10.1              # via tox
-tox-battery==0.6.0        # via -r requirements/dev.in
```

Comment:
Please pin tox-battery library to 0.5.2 in the `constraints.txt` file instead of changing it here.